docs/documentation/upgrading/topics/keycloak/changes-22_0_0.adoc

```diff
@@ -74,3 +74,13 @@ NOTE:: When the auto-build runs, the build time options will be in effect for al
 In previous releases the `export` and `import` commands allowed runtime parameters like, for example, a database URL only in configuration files or environment variables.
 Starting with this release, those runtime parameters are now available on the command line as well.
 Use the `--help` option to find out about the supported parameters.
+
+# Consistent fallback message resolving for all themes
```

Comment:
This is not a Markdown file but AsciiDoc, so I believe you need to use `=` for titles (like in the rest of this file).
```suggestion
= Consistent fallback message resolving for all themes
```

Reply:
Thanks for the hint, adjusted that.